cmd/check_container.go

```diff
@@ -229,3 +229,16 @@ func init() {
 
 	checkCmd.AddCommand(checkContainerCmd)
 }
+
+func buildConnectURL(projectID string) string {
+	connectURL := fmt.Sprintf("https://connect.redhat.com/projects/%s/overview", projectID)
```

Comment:
maybe it's worth moving urls and the like into a constants.go ?
would be easier to maintain later.

Reply:
This URL is only for display purposes to the user, we do not call this URL in the code. The `catalog` URL is in `defaults`, I'm not sure we need to add `connect` there as well.